tools/backups2datalad-cron

```diff
@@ -12,18 +12,20 @@ ds=$(dirname "$ds")
 
 source ~/.bashrc-miniconda
 conda activate dandisets
+pip install -r backups2datalad.req.txt
 
 cd "$ds"
 export DATALAD_LOG_LEVEL=WARNING  # otherwise too much of noise
 
 set -x
 eval tools/backups2datalad.py \
-   --backup-remote dandi-dandisets-dropbox  \
-   --gh-org dandisets \
    -l WARNING \
    -J 5 \
-   "$girderdir" \
-   "$ds" \
+   --assetstore "$girderdir" \
+   --target "$ds" \
+   update-from-backup \
```

Comment:
`update-from-backup` (or a less preferable separate invocation in the cron job script) should also take care about all published versions of the dandiset

<details>
<summary>update-from-backup seems to not do that on my attempt for 000029:</summary> 

```shell
(dandisets) dandi@drogon:/tmp/dandisets$ DATALAD_LOG_LEVEL=WARNING tools/backups2datalad.py --assetstore $girderdir -l INFO  -J 1  -T . update-from-backup 000029
2021-08-09T14:14:45-0400 [INFO    ] backups2datalad.py Syncing Dandiset 000029
2021-08-09T14:14:45-0400 [INFO    ] backups2datalad.py Updating metadata file
nothing to save, working tree clean
nothing to save, working tree clean
2021-08-09T14:14:45-0400 [INFO    ] backups2datalad.py No changes made to repository; deleting logfile
(dandisets) dandi@drogon:/tmp/dandisets$ git -C 000029 tag
(dandisets) dandi@drogon:/tmp/dandisets$ git -C 000029 show | head
commit 3bef25dd90463d5da6dc48733bb7623048e067b6
Author: DANDI Meta-user <dandi@mit.edu>
Date:   Fri Aug 6 17:22:47 2021 -0400

    [backups2datalad] only some metadata updates

diff --git a/.dandi/assets.json b/.dandi/assets.json
index 26c54cf..f55ccc9 100644
--- a/.dandi/assets.json
+++ b/.dandi/assets.json

```
</details>

that was running of a Merge: b9aaa11 (master) and 38b96c4 (this PR)

Reply:
I think this happened because `tag_releases()` is only called if `sync_dataset()` indicates that any changes were made to the local copy of the Dandiset.

Reply:
Let's add an option to force that e.g. `--force-versions-check` or alike?

Reply:
@yarikoptic What about instead always ensuring that the GitHub remote & tags exist and having the changes check only control whether anything gets pushed?

Reply:
IMHO having tags on a remote (GitHub) is complementary to having tags locally and better test against local. For unit testing we will not even push to github

Reply:
@yarikoptic The "GitHub remote" and pushing parts would only take effect if `gh_org` is set, which it is not while testing.

Reply:
Correct, I must be missing connection to the need to ensure having tags for versions even if no new changes to draft were detected since the last run

Reply:
Note: creating a new tag, while no changes to draft, also warrants a push if gh_org is set

Reply:
The code now always syncs tags even if no changes were made to the dataset.

Reply:
Great, did you test it out on 000029?

Reply:
@yarikoptic No.  How exactly should I run this script?  I can't run it in a fresh environment, as the dataset needs to contain commits from prior script runs.

Reply:
the same as I did in the original comment in [this thread above]( https://github.com/dandi/dandisets/pull/55#discussion_r685420050)   - on a git clone of the `dandisets` somewhere else and without `--gh-org` so nothing is pushed, so we are safe.  There is still a `screen` session already on `dandi@drogon` you could use, just merge recent changes in this PR

Reply:
@yarikoptic Tested.  Seems to work.

Reply:
It works indeed, awesome!  I will post follow up comments in a separate thread